### DISCUSSION TITLE
Wordpress CLS

### DISCUSSION THREAD

**Discussion by akmal992310** [community/community#38080] [Other Feature Feedback, Questions, & Ideas]
Hi ! I'm a newbie on GITHUB. I create this account just to ask some questions about Cumulative Layout Shift (CLS) in WORDPRESS. Can anyone help me to solve this issues. Thank you so much !
<img width="919" alt="Screenshot 2022-11-03 at 12 05 02 AM" src="https://user-images.githubusercontent.com/117299677/199541317-79c1b699-e889-4a9d-b066-89a7db52fccb.png">

**Comment by CharlesRakesh** [comment]
https://github.com/CharlesRakesh/Elementor-Pro-Form/issues/1
